README.md

```diff
@@ -1 +1,87 @@
-# covid-tracker
\ No newline at end of file
+
+![](https://img.shields.io/static/v1?label=BY&message=Reemoz&color=pink)
+
+<!-- Feel Free to Add, Update, Delete Any Section you find needs so -->
+
+# Covid19-Tracker 🦠
+
+> This is My react capstone project, This project is talking about covid cases in all countries in the world and also displays all the information details to each country with all covid cases or recovered cases.
+
+
+<!-- ## Video Demo 📽 -->
+
+
+## Deploy Demo 🚀
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/cc500adb-5279-4240-9ff1-26be38ac6130/deploy-status)](https://covid19-tracker9.netlify.app/)
+
+## Video Description by me
+
+[see my video from here](https://www.loom.com/share/abd3e97140784135bbc8ddbf2906bb86)
+
+
+
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm install`
+
+### `npm start`
+
+
+
+## Built With
+
+- Languages: _**HTML, CSS, JavaScript, Reactjs, Redux**_
+- Frameworks: _**N/A**_
+- Technologies used: _**GIT, GITHUB, LINTERS**_
+
+## Additional tools
+ - Google fonts
+ - Webpack 
+ - React Icons
+ - linters
+ - Gitflow
+ - Redux
+ - Test by jest
+
+
+## Authors
+
+<!-- Only Change Username for Different Accounts -->
+
+👤 **Reem**
+
+ Platform | Badge |
+ --- | --- |
+ **GitHub**  | [@Reem-lab](https://github.com/Reem-lab)
+ **Twitter** | [Rem79940127](https://twitter.com/Rem79940127)
+ **LinkdIn** | [reem-janina](https://www.linkedin.com/in/reem-janina-ab74ab21a/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/MrRamoun/WEBDEV/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
```

Comment:
- [ ] It would be better if you give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

![Capture](https://user-images.githubusercontent.com/64173189/157733518-f887cdb9-ec9b-4af7-8acb-35c99a691b31.JPG)

Reply:
Thanks @BrianSammit  😊 I will add it